src/kakarot/accounts/library.cairo

```diff
@@ -228,71 +221,9 @@ namespace AccountContract {
         let pedersen_ptr = cast([ap - 2], HashBuiltin*);
         let range_check_ptr = [ap - 1];
```

Comment:
what do you think about 
```
        // Whitelisting pre-eip155
        let (is_authorized) = Account_authorized_message_hashes.read(msg_hash);
        if (pre_eip155_tx != FALSE) {
            with_attr error_message("Unauthorized pre-eip155 transaction") {
                assert is_authorized = TRUE;
            }
        }
```
It avoids the big if and assigning the pointers
Drawback is that we are using a reference before its usage in the if